mozperftest_tools/tools/side_by_side.py

```diff
@@ -0,0 +1,861 @@
+#!/usr/bin/python3
+"""
+Used to produce comparisons of browsertime videos between a base
+and a new revision.
+"""
+import cv2
+import gc
+import numpy as np
+import os
+import pathlib
+import json
+import shutil
+import subprocess
+
+from matplotlib import pyplot as plt
+from scipy.stats import spearmanr
+
+try:
+    from urllib.parse import urlencode
+    from urllib.request import urlopen, urlretrieve
+except ImportError:
+    from urllib import urlencode, urlretrieve
+    from urllib2 import urlopen
+
+from artifact_downloader import artifact_downloader
```

Comment:
Where are these files coming from? I don't see them in the folder and I think they are being found in the top-level directory.